src/config/validate.ts

```diff
@@ -101,6 +102,13 @@ function validateUrlLifetime(urlLifetime: string): void {
 	}
 }
 
+function validateUrlExpireFrom(urlExpire: string): void {
+	logger.debug(`Start validateUrlExpireFrom with ${urlExpire}`)
+	if (!urlExpire || (urlExpire != 'create' && urlExpire != 'update')) {
```

Comment:
The feature we talked about is "expire from either 'create' or 'last used' ". You should add the 'last used'.